docusaurus.config.js

```diff
@@ -226,6 +252,11 @@ const config = {
             label: "Blog",
             position: "left",
           },
+          {
+            to: "/community/",
+            label: "Community",
+            position: "left",
+          },
```

Comment:
Can we move this block so it sits right after `Docs`?

Reply:
Sure thing!